src/helpers/util.js

```diff
@@ -376,6 +376,54 @@ class Util {
       glob(pattern, options, (error, files) => error ? reject(error) : resolve(files))
     );
   }
+
+  /**
+   * Compose AWS credentials string
+   * @param {Object} accountData
+   * @param {Object} [sourceProfile]
+   * @param {Boolean} tfvars
+   * @return {String}
+   */
+  static prepareCredentialsFile({ accountData, sourceProfile, tfvars = false }) {
+    let credentials = `[default]\n`;
+
+    if (sourceProfile) {
+      credentials += `aws_access_key_id = ${sourceProfile.env_var.AWS_ACCESS_KEY_ID.value}\n` +
+        `aws_secret_access_key = ${sourceProfile.env_var.AWS_SECRET_ACCESS_KEY.value}\n` +
+        `role_arn = ${accountData.env_var.AWS_ROLE_ARN.value}\n`;
+    } else {
+      credentials += `aws_access_key_id = ${accountData.env_var.AWS_ACCESS_KEY_ID.value}\n` +
+        `aws_secret_access_key = ${accountData.env_var.AWS_SECRET_ACCESS_KEY.value}\n`;
+    }
+
+    if (tfvars) {
+      credentials += `output = json\n` +
+        `region = us-east-1\n`;
```

Comment:
region might also be passed as `AWS_DEFAULT_REGION` variable, so please update with condition `if ${accountData.env_var.AWS_DEFAULT_REGION.value} then use env var else us-east-1`